config/initializers/instrumentation_notification.rb

```diff
@@ -21,6 +21,7 @@ def process_action(*args)
         params: request.filtered_parameters,
         format: request.format.try(:ref),
         method: request.method,
+        tags: YetiConfig.api_log_tags.presence || [],
```

Comment:
There you have to add not tags defined in **api_log_tags**(which is all possible tags) but tags from other variable, like **api_log_add_tags**. In this way it will be possible to add different tags on  different application instances

Reply:
so to implement this suggestion I want to know how to fill in this additional variable/method

here example of usage

```
def process_action(*args)
      return super unless request.path.start_with?('/api/')

      raw_payload = {
        controller: self.class.name,
        action: action_name,
        meta: meta,
        params: request.filtered_parameters,
        format: request.format.try(:ref),
        method: request.method,
        tags: api_log_add_tags,
        path: (begin
                      request.fullpath
               rescue StandardError
                 'unknown'
                    end)
      }
```
Would you like to define the api_log_add_tags method in these controllers?

```
          app/controllers/api/cryptomus_callbacks_controller.rb
          app/controllers/api/rest/admin/auth_controller.rb
          app/controllers/api/rest/customer/v1/auth_controller.rb
          app/controllers/api_controller.rb
```

or you want to do define a separate config section (`api_log_add_tags `) inside Yeti Web yml config file

Reply:
use case:

- Two instances of application
- Instance A, should add tagA to logs
- Instance B, should add tagB to logs
- When admin checking logs in UI he should be able to filter logs by tagA and tagB - it means UI should know all possible tags to build list in filter.

solution1:
- two config variables
- **api_log_tags** - to define all possible tags(will be used to build list in filters in UI). it will have same value on both instances( [tagA, tagB].
- **api_log_add_tags** - to define which tag will be added during writing API logs. it will have different values on each instance

solution2:
- one config variable
- no tag list in filter UI, just text input to allow enter any tag name manually. It is admin responsibility to know which value to enter
- one config variable to define which tag will be added during writing API logs. it will have different values on each instance

Reply:
@dmitry-sinina @Ivanov-Anton solution2 is more simple. I suggest to implement it